### DISCUSSION TITLE
[Feature] Better support for 1Password in the App

### DISCUSSION THREAD

**Discussion by AleksCee** [immich-app/immich#10912] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi,
I use 1PWD as passwordmanager and in the app it didn’t finde automatically the item when I choose autofill on iOS and whene I manually select the right item is not work by the first try. I musst change the login field from email to password or from password to email, sometimes then the values were passed but only after the second try. Other apps can get the right item in 1PWD without manually searching for, so I think there is a way to improve this?
Thanks, Alex

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5177

**Reply by AleksCee** [reply]
Thanks - is this the same issue in Android and iOS?
